docs-src/content/protobuf/_index.md

```diff
@@ -350,6 +353,60 @@ current state.
 
 
 
+<a name="TranscribeAction"></a>
+### Message: TranscribeAction
+This action indicates that the client application should
+transcribe the user's input.
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| id | [string](#string) |  | <p>The ID of the transcribe action, which is useful to differentiate separate transcription tasks within a single sesssion.</p> |
+| cubic_model_id | [string](#string) |  | <p>The ASR model to use for transcription.</p> |
+| diatheke_model_id | [string](#string) |  | <p>The Diatheke model where this transcribe action is defined. If empty, the server will not be able to automatically close the transcribe stream based on conditions defined in the Diatheke model, such as a non-speech timeout or an "end-transcription" intent. When empty, the stream must be closed by the client application.</p> |
+
+
+
+
+
+
+
+<a name="TranscribeInput"></a>
+### Message: TranscribeInput
+Data to send to the Transcribe stream. The first message on
+the stream must be a TranscribeAction, followed by audio data.
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| action | [TranscribeAction](#TranscribeAction) |  | <p>Action defining the transcribe configuration.</p> |
+| audio | [bytes](#bytes) |  | <p>Audio data to transcribe.</p> |
+
+
+
+
+
+
+
+<a name="TranscribeResult"></a>
+### Message: TranscribeResult
+The result from the Transcribe stream. Usually, many partial
+(or intermediate) transcriptions will be sent until the final
+transcription is ready for every utterance processed.
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| text | [string](#string) |  | <p>The transcription.</p> |
+| confidence | [double](#double) |  | <p>Confidence estimate between 0 and 1. A higher number represents a higher likelihood of the transcription being correct.</p> |
+| is_partial | [bool](#bool) |  | <p>True if this is a partial result, in which case the text of the transcription for the current utterance being processed is allowed to change in future results. When false, this represents the final transcription for an utterance, which will not change with further audio input. It is sent when the ASR has endpointed. After the final transcription is sent, any additional results sent on the Transcribe stream belong to the next utterance.</p> |
```

Comment:
```suggestion
| is_partial | [bool](#bool) |  | <p>True if this is a partial result, in which case the next result will be for the same audio, either repeating or correcting the text in this result. When false, this represents the final transcription for an utterance, which will not change with further audio input. It is sent when the ASR has identified an endpoint. After the final transcription is sent, any additional results sent on the Transcribe stream belong to the next utterance.</p> |
```

Reply:
Also auto-generated. I suppose I can go back an update the protobuf file and regenerate.